app/src/main/java/org/xbmc/kore/ui/generic/NavigationDrawerFragment.java

```diff
@@ -81,9 +82,11 @@
             ACTIVITY_TVSHOWS = 3,
             ACTIVITY_MUSIC = 4,
             ACTIVITY_PVR = 5,
-            ACTIVITY_FILES = 6,
-            ACTIVITY_ADDONS = 7,
-            ACTIVITY_SETTINGS = 8;
+            ACTIVITY_FAVOURITES = 6,
+            ACTIVITY_FILES = 7,
+            ACTIVITY_ADDONS = 8,
+            ACTIVITY_SETTINGS = 9;
+
```

Comment:
I went with your first suggestion because it felt simpler, and I was afraid that I may introduce other bugs if I went with the second approach.